index.html

```diff
@@ -13,14 +13,44 @@
     />
     <link rel="stylesheet" href="css/style.css" />
     <script src="./js/main.js" defer></script>
+    <script src="luxon.js" defer></script>
   </head>
   <body>
-    <div class="container my-4 max-width-700">
-      <h1 class="display-4 text-center">
+    <nav
+      class="
+        d-flex
+        justify-content-between
+        align-items-center
+        my-4
+        max-width-800
+      "
+    >
+      <h1 class="display-6">
         All Awesome <span class="text-primary">Books</span>
       </h1>
-
-      <table class="table table-striped mt-5">
+      <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a id="list-view" class="nav-link" href="#">List</a>
+        </li>
+        <li class="nav-item">
+          <a
+            id="add-view"
+            class="nav-link active"
+            aria-current="page"
+            href="index.html"
+            >Add New</a
+          >
+        </li>
+        <li class="nav-item">
+          <a id="contact-view" class="nav-link" href="#">Contact</a>
+        </li>
+      </ul>
```

Comment:
- [ ] The right menu elements in your navbar is missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.
- [ ] To enhance the user experience of this design kindly ensure that an active menu link is highlighted (should have color change or change in opacity  of the active menu item). In other words, the current active page should have the corresponding menu item highlighted.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/129295547-d3626838-0b1d-4533-9baf-9ed5a80ac085.png)

**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)